### DISCUSSION TITLE
How to deal with the missing data in the gauges station?

### DISCUSSION THREAD

**Discussion by LuvMangorange** [mhm-ufz/mHM#29] [Q&A] [answered]
Hi guys
There ara some missing values in the observation data from 2002 to 2013, how to deal with them?
Thanks for help!

**Comment by rohinikumar82** [comment, marked as answer]
This you can specify as in the gauging station discharge text file (see header info. in the file)... Currently in the file for the test basin 01: nodata is set to -9999.0 (you can also use any other value like -999 or -1e+32 or... but you need to specify this in the header info of the discharge text file). 

Here is snap shot the discharge file: see the second line "nodata   -9999" 

-------------
00333 Gauge 2 (daily discharge)
nodata   -9999
n       1       measurements per day [1, 1440]
start  1988 01 01 00 00   (YYYY MM DD HH MM)
end    1995 12 31 00 00   (YYYY MM DD HH MM)
1988  01  01  00  00   1500.000
1988  01  02  00  00   1440.000

--------------------

Best,
Rohini

**Reply by LuvMangorange** [reply]
Hi @rohinikumar82 
Thank you! 
The problem is solved.
